docs/device-management/1-status/README.md

```diff
@@ -0,0 +1,57 @@
+# Status
+
+Device status is described by three attributes, all of which are visible when
+viewing a device summary in the Golioth console.
+
+- **Session Established**: the time at which a device last [established a secure
+  session](/device-management/authentication) with the Golioth platform.
+- **Last Report**: the time at which a device last successfully delivered any
+  data to the Golioth platform.
+- **Setting Sync**: the time at which a device last reported the status of its
+  settings, and whether they were synchronized with the values set on the
+  Golioth platform.
+
+## Analyzing Device Status
+
+The three device status attributes can be used in concert to analyze different
+asepcts of a device's behavior. Examples for how to interpret various
+permutations of device status attributes are provided below.
+
+### Handshake Frequency
+
+It is typically optimal to maintain a session with the Golioth platform as long
+as possible, as re-establishing a session uses network bandwidth, power, and
+compute resources for a device. Approxmate session duration can be ascertained
+by calculating the difference between the **Session Established** and **Last
+Report** attributes.
+
+For example, if the **Session Established** timestamp is 20 hours old, but the
+**Last Report** timestamp is a few seconds ago, the device has maintained the
+same session of approximately 20 hours. In contrast, if the **Session
+Established** timestamp and **Last Report** timestamp are frequently close in
+value, it is potentially a sign that the device is performing unnecessarily
+frequent handshakes.
```

Comment:
```suggestion
frequent DTLS handshakes.
```

Reply:
I think I'd prefer to keep this agnostic to protocol for the time-being. I don't think we lose too much information by excluding "DTLS" here, and this could also technically be impacted by a TLS handshake when using HTTP with the same credentials (uncommon at the moment).